docs/governance/steering/elections.md

```diff
@@ -0,0 +1,129 @@
+# Common Cloud Controls (CCC) Steering Committee Elections
+
+This document outlines the process for steering committee elections.
+
+## Eligibility
+
+Restrictions on candidacy and voting are in place to ensure that the CCC project is led by individuals who are highly trusted, impactful, and currently active in the community.
+
+### Eligibility for voting
+
+In order to be eligible for voting, an individual must be:
+
+- A FINOS Member<sup>[1](#footnote1)</sup> who had a substantial contributions 
```

Comment:
a substantial contribution
substantial contributions
Either of these two is fine but not "a substantial contributions"!